crates/rustfix/Readme.md

```diff
@@ -5,31 +5,12 @@
 
 Rustfix is a library defining useful structures that represent fix suggestions from rustc.
 
-## Current status
+This is a low-level library. You pass it the JSON output from `rustc`, and you can then use it to apply suggestions to in-memory strings. This library doesn't execute commands, or read or write from the filesystem.
 
-Currently, rustfix is split into two crates:
+If you are looking for the [`cargo fix`] implementation, the core of it is located in [`cargo::ops::fix`].
 
-- `rustfix`, a library for consuming and applying suggestions in the format that `rustc` outputs (this crate)
-- `cargo-fix`, a binary that works as cargo subcommand and that end users will use to fix their code (maintained in the [cargo](https://github.com/rust-lang/cargo/blob/master/src/bin/cargo/commands/fix.rs) repo).
-
-
-The library (and therefore this repo) is considered largely feature-complete. This is because:
-* There is no compiler or even rust-specific logic here
-* New lints and suggestions come from the Rust compiler (and external lints, like [clippy]).
-* `rustfix` doesn't touch the filesystem to implement fixes, or read from disk
```

Comment:
[not a blocker]
This can be put somehere either in rustfix's lib.rs or somewhere in design architecture.

Reply:
I put the beginning of a doc-comment in the library. My intention is to defer doing documentation work after this merged. There is essentially no documentation, and it can be a bit much to review updating all of that independently.